SqlAdmin/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "google/cloud-sql-admin",
+    "description": "Google Cloud Sql Admin Client for PHP",
+    "license": "Apache-2.0",
+    "minimum-stability": "stable",
+    "autoload": {
+        "psr-4": {
+            "Google\\Cloud\\Sql\\": "src",
```

Comment:
Likewise, the composer package name here is inconsistent with the PHP namespace. I would expect the namespace to be `Google\\Cloud\\Sql\\Admin\\` or the package to be `google/cloud-sql`. Changing the namespace in [the proto](https://github.com/googleapis/googleapis/blob/master/google/cloud/sql/v1beta4/cloud_sql.proto#L26) would lead to inconsistencies between languages, so it's probably not a good idea. 

I want to make sure this is a conscious decision to have this discrepancy, and what the justification is.

Reply:
Good call out. It was sql, but then I noticed nodejs is using https://github.com/googleapis/nodejs-sql-admin already so I matched it for consistency.